.cbmc-batch/jobs/aws_cryptosdk_md_update/README.md

```diff
@@ -0,0 +1,3 @@
+# Memory safety proof for aws_cryptosdk_md_update
+
+This proof harness attains 100% code coverage.
```

Comment:
See previous comments. You can remove these README file.

Reply:
Done!